docs/lob_guide/src/asciidoc/_chapters/introduction.adoc

```diff
@@ -0,0 +1,152 @@
+////
+/**
+* @@@ START COPYRIGHT @@@
+*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements.  See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership.  The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License.  You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*
+* @@@ END COPYRIGHT @@@
+*/
+////
+
+[#introduction]
+= Introduction
+
+[#what is lOB]
+== What is LOB
+
+LOB (Large Object), a set of large object data types used to store large volumes of data, provides random and piece-wise access to the data. 
+
+Database support for LOB is not universal.
+
+[#lob data types]
+== LOB Data Types
+
+The following are the data types supported by Trafodion. +
+
+
+[caption="Table 1-1 "]
+.Descriptions for BLOB and CLOB
+[cols="30%,70%",options="header"]
+|===
+| *Data Type*         | *Desciprtion*
+| BLOB                | Binary Large Object. + 
+
+Holds large blocks of unstructured data in binary format. +
+
+BLOB is ideal for storing and handling unstructured data, such as images, audio, video and other multimedia objects.
+| CLOB                | Character Large Object. +
+
+Stores unusually large amounts of string data in character set format. +
+
+CLOB is ideal for storing and processing semi-structured data, such as large document or string.
+|===
+
+[#lob storage]
+== LOB Storage
+
+The basic design idea is to allow a database row to have multiple LOB columns and a table to have multiple such rows. The design takes a combination approach of rows store and column store. 
+
+When a SQL table with LOB column is created, there are several dependent objects that are created to hold metadata information on the LOB as well as the data.
+
+The LOB data itself is stored in HDFS while the Trafodion table contains a unique LOB handle for each row/column. The LOB handle provides the identifier that is used to query the LOB metadata tables to find the location of the LOB data files.

```

Comment:
"an HDFS file" is better. A question: Is each LOB value stored in a separate HDFS file? Or are several LOB values stored in one HDFS file? Also, "...a unique LOB handle for each row/column" might be better stated, "...a unique LOB handle for each LOB value".

Reply:
Thanks Dave.
I've corrected. :)

Answer from Sandhya:
Each lob column in any table will have  it’s own unique lob data file. So any lob data belonging to that column will be stored in that HDFS file. 
So if a table has 3 columns, you will see 3 lob data files with the first part of the name being the same and the column number will be appended to it.

For example, for a table with 2 lob columns:
Location: /user/trafodion/lobs
DataFile: LOBP_04474425229029907479_0001

Location: /user/trafodion/lobs
DataFile: LOBP_04474425229029907479_0002